zipline/algorithm.py

```diff
@@ -2384,3 +2398,9 @@ def all_api_methods(cls):
 
 # Map from calendar name to default domain for that calendar.
 _DEFAULT_DOMAINS = {d.calendar_name: d for d in domain.BUILT_IN_DOMAINS}
+# Map from calendar name to default country code for that calendar.
+_DEFAULT_FETCH_CSV_COUNTRY_CODES = {
```

Comment:
Maybe this could just be called `_DEFAULT_CALENDAR_COUNTRY_CODES` or somesuch, since it isn't necessarily tied to `fetch_csv`.

Reply:
I'm not sure that the choice I'm making here would be the right choices for any other API that needed a default country code. I think it **probably** would be, but since nothing else is using this I don't want to make this look more general than it is until we're confident it can be used generically.

Reply:
Ok makes sense.